### DISCUSSION TITLE
Add a Talked server for call recording

### DISCUSSION THREAD

**Discussion by funbotan** [nextcloud/all-in-one#552] [Ideas] [answered]
[Talked](https://apps.nextcloud.com/apps/talked) is currently the only option for server-side call recording on NC Talk. However, it requires a [server](https://github.com/MetaProvide/talked), which is a pain to set up. It would be great to include this service in the AIO.

**Comment by szaimen** [comment, marked as answer]
I fear this does not pass all requirements for integrating this new container into AIO.
See https://github.com/nextcloud/all-in-one#requirements-for-integrating-new-containers

**Reply by szaimen** [reply]
However you might be interested in this: #1581

**Comment by nr-plaxon** [comment]
Would you consider the "official" spreed-bases recording service for integration in Nextcloud AIO?
[https://github.com/nextcloud/spreed/pull/9177](https://github.com/nextcloud/spreed/pull/9177)

**Reply by szaimen** [reply]
Can you create a new feature request for this? See https://github.com/nextcloud/all-in-one/discussions/new?category=ideas

**Comment by funbotan** [comment]
Since [NC Talk developers have announced](https://nextcloud.com/blog/nextcloud-talk-in-hub-4-next-level-video-chat/) that they will integrate call recording out of the box, I would consider this issue closed, at least for my use case.
